content/docs/command-reference/fetch.md

```diff
@@ -77,6 +79,9 @@ or `-T` options are used).
   [remote storage](/doc/command-reference/remote) to fetch from (see
   `dvc remote list`).
 
+- `--run-cache` - downloads all available history of stage runs from the remote
+  repository.
```

Comment:
Is there a way to know how big this is? Or to list all the checkpoints that have been saved between each commit or with timestamps, or something like that? Thanks

Reply:
Nope, it is pretty premitive for now.